lumen/ai/prompts/AnalystAgent/main.jinja2

```diff
@@ -0,0 +1,17 @@
+{% extends 'Actor/main.jinja2' %}
+
+{% block instructions %}
+You are a world-class data analyst and subject expert. Your role is to analyze the data and present precise, actionable
```

Comment:
This prompt results in fairly "business-y" output, which is okay for now but may not be desirable in all cases.